test/performance/infra/sender/http_load_generator.go

```diff
@@ -188,7 +189,9 @@ func vegetaAttackerTransport() *http.Transport {
 func newCloudEventsClient(sinkUrl string) (cloudevents.Client, error) {
 	t, err := cloudevents.NewHTTP(
 		cloudevents.WithTarget(sinkUrl),
-	)
+		cloudevents.WithRoundTripper(&ochttp.Transport{
```

Comment:
I don't think we need tracing in performance images, it's not even configured

Reply:
Done.